frontend/src/global_styles/layout/work_packages/_table_baseline.sass

```diff
@@ -46,7 +46,7 @@
     padding: 2px 2px 0 2px
 
   &--cell
-    background-color: #f3f3f3
+    background-color: var(--bgColor-muted)
```

Comment:
The contrast is too low, imho. Even in the light mode, you barely see the fields, and in dark mode it is even worse

<img width="1000" alt="Bildschirmfoto 2024-07-12 um 11 22 06" src="https://github.com/user-attachments/assets/e77bd9ef-f2db-45f2-b1ea-c96774b1eb18">

<img width="1000" alt="Bildschirmfoto 2024-07-12 um 11 22 52" src="https://github.com/user-attachments/assets/4a32d658-0dda-4c73-8b31-2f038d17a1f4">